cmd/openscap-plugin/xccdf/datastream.go

```diff
@@ -160,6 +168,36 @@ func populateProfileVariables(dsProfile *xmlquery.Node, parsedProfile *xccdf.Pro
 	return parsedProfile, nil
 }
 
+func populateProfileRules(dsProfile *xmlquery.Node, parsedProfile *xccdf.ProfileElement) (*xccdf.ProfileElement, error) {
+	if parsedProfile.Selections == nil {
+		parsedProfile.Selections = []xccdf.SelectElement{}
+	}
+	profileRules, err := getDsElements(dsProfile, "xccdf-1.2:select")
```

Comment:
```suggestion

	profileRules, err := getDsElements(dsProfile, "xccdf-1.2:select")
```
Nit, but since it is very similar to `populateProfileVariables` it would be nice to keep the empty lines consistent.